docs/Tutorials/Examples/Private-Network-Example.md

```diff
@@ -4,37 +4,45 @@ description: Quorum dev quickstart tutorial
 
 # Quorum Developer Quickstart tutorial
 
-The Quorum Developer Quickstart uses the Hyperledger Besu Docker image to run a private network of Besu
-nodes managed by Docker Compose.
+The Quorum Developer Quickstart uses the Hyperledger Besu Docker image to run a private
+[IBFT 2.0](../../HowTo/Configure/Consensus-Protocols/IBFT.md) network of Besu nodes managed by Docker Compose.
 
-!!! important
-
-    This tutorial runs a private network suitable for education or demonstration purposes and is
-    not intended for running production networks.
+!!! warning
+    This tutorial runs a private network suitable for education or demonstration purposes
+    and is not intended for running production networks.
 
 ## Prerequisites
 
-* [Docker and Docker-compose](https://docs.docker.com/compose/install/)
-* [Nodejs](https://nodejs.org/en/download/)
-* On Windows:
-    * Windows Subsystem for Linux 2
+* One of the following operating systems is required:
+    * Linux on x86_64 architecture
+    * macOS on an Intel processor (M1 processor not supported yet)
+    * Windows 64-bit edition
+* [Docker and Docker-compose](https://docs.docker.com/compose/install/).
+* On Windows only you will need:
+    * Windows Subsystem for Linux 2.
     * Docker desktop configured to use the WSL2-based engine.
```

Comment:
Could this be level 3 bullets below "Windows 64-bit edition". So it could be, "Windows 64-bit edition, with:"